cfg/params.yaml

```
@@ -346,6 +346,49 @@
   usage: Param for exponential backoff algorithm, which is used to increase waiting time b/w two consecutive retries.
   default: 2
 
+- config-path: "gcs-retries.read-stall.enable"
+  flag-name: "enable-read-stall-retry"
+  type: "bool"
+  usage: >-
+    Enable/disable retry for stalled read-request based on dynamic timeout.
```

">" is used for breaking a long string into multiple lines and is not needed here. The "-" at the end removes the newline in multiline strings that gets added automatically. They are not needed here since it's a short string.
```suggestion
  usage: Enables retry for stalled read-request based on dynamic timeout.
```

---

Added more detailed description, which requires newline.